### PR TITLE
Added subtract functionality

### PR DESCRIPTION
**Pull request by MichaelTamaki**
Added docstring, functionality, and test.

### CODE REVIEW

File: calc/tests/test_calc.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
-from calc import add
+from calc import add, subtract
 
 def test_add():
     assert add(1, 2) == 3
 
+def test_sub():
+    assert subtract(2, 1) == 1
```
Comment (sirlittle) [inline calc/tests/test_calc.py:8]: Maybe add a few more tests to check functionality especially with negative numbers.

File: calc/__init__.py [hunk 2]
```diff
@@ -5,7 +5,16 @@ def add(a, b):
     return a - b
 
 def subtract(a, b):
-    return 0
+    """ Subtract 2 integers.
```
Comment (stefanv) [inline calc/__init__.py]: Why only integers?

File: calc/tests/test_calc.py (hunk 1 above)
Comment (stefanv) [reply calc/tests/test_calc.py:8]: :+1:

**Comment by stefanv** [comment]
Please rebase on master
